lib/dealership.rb

```diff
@@ -0,0 +1,58 @@
+require './lib/car'
+class Dealership
+  attr_reader :inventory
+
+  def initialize(name, address)
+    @name = name
+    @address = address
+    @inventory = []
+  end
+
+  def inventory_count
+    @inventory.size
+  end
+
+  def add_car(car)
+    @inventory << car
+  end
+
+  def has_inventory?
+    @inventory.size > 0 ? true : false
+  end
```

Comment:
Since @inventory.size > 0 returns a boolean you could shorten this even further

```suggestion
  def has_inventory?
    @inventory.size > 0
  end
```